packages/ods/src/components/checkbox/src/components/ods-checkbox/ods-checkbox.tsx

```diff
@@ -184,7 +185,7 @@ export class OdsCheckbox {
           aria-labelledby={ this.ariaLabelledby }
           class={{
             'ods-checkbox__checkbox': true,
-            'ods-checkbox__checkbox--error': !!this.isInvalid,
+            'ods-checkbox__checkbox--error': this.hasError || !!this.isInvalid,
```

Comment:
Can we have the checkable component in error and checked ? Because the border red is not visible when the component is checked

Reply:
Not in a way I can think of, if those are checked, they shouldn't be in error anymore